modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -33,9 +33,9 @@ Templates do not include the authentication credentials that are specified for t
 
 == Procedure
 
-. Select *Publish Template* from the drop-down menu which is in the top-right corner of the Flow Designer screen.
+. Click the *Publish* button, which is in the top-right corner of the Flow Designer screen.
```

Comment:
Click *Publish*  from...

Reply:
Should confirm in the style guide moving forward.